apbs/test/test_pqr/test_pqrreader.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+from apbs.chemistry.atom_list import AtomList
+from apbs.pqr import PQRReader
+
+
+class TestPQRReader:
+    def test_ctor(self):
+        sut = PQRReader()
+        sample = r"""
+ATOM   5226  HD1 TYR   337     -24.642  -2.718  30.187  0.115 1.358
+ATOM      7  CD   LYS D   1      44.946 37.289  9.712    -0.0608  1.9080
+ATOM     39 O3PB ADP     1     -16.362  -6.763  26.980 -0.900 1.700
+ATOM     39 O3PB ADP     1  D   -16.362  -6.763  26.980 -0.900 1.700
+"""
+        atomlist: AtomList = sut.loads(sample)
+        for idx in range(1):
+            assert abs(atomlist.atoms[idx].x - -24.642) < 0.001
+            assert int(atomlist.atoms[idx].x) == -24
+
+    def test_load(self):
+        sut = PQRReader()
+        pqr_imput = (
+            pathlib.Path(__file__).parent.parent.parent.parent.absolute()
+            / "examples/actin-dimer/mol1.pqr"
```

Comment:
If windows makes up such a large part of our userbase, perhaps we should prefer `os.path.join(...)`.

Reply:
@sobolevnrm prefers that we use pathlib instead of os.path.